internal/mode/static/manager.go

```diff
@@ -419,25 +444,65 @@ func createTelemetryJob(
 	logger := cfg.Logger.WithName("telemetryJob")
 	exporter := telemetry.NewLoggingExporter(cfg.Logger.WithName("telemetryExporter").V(1 /* debug */))
 
-	worker := telemetry.CreateTelemetryJobWorker(logger, exporter, dataCollector)
-
-	// 10 min jitter is enough per telemetry destination recommendation
-	// For the default period of 24 hours, jitter will be 10min /(24*60)min  = 0.0069
-	jitterFactor := 10.0 / (24 * 60) // added jitter is bound by jitterFactor * period
-
 	return &runnables.Leader{
 		Runnable: runnables.NewCronJob(
 			runnables.CronJobConfig{
-				Worker:       worker,
+				Worker:       telemetry.CreateTelemetryJobWorker(logger, exporter, dataCollector),
 				Logger:       logger,
 				Period:       cfg.TelemetryReportPeriod,
-				JitterFactor: jitterFactor,
+				JitterFactor: telemetryJitterFactor,
 				ReadyCh:      readyCh,
 			},
 		),
 	}
 }
 
+func createUsageReporterJob(
+	k8sClient client.Reader,
+	cfg config.Config,
+	usageSecret usage.SecretStorer,
+	readyCh <-chan struct{},
+) (*runnables.Leader, error) {
+	logger := cfg.Logger.WithName("usageReporter")
+	reporter, err := usage.NewNIMReporter(
+		usageSecret,
+		cfg.UsageReportConfig.ServerURL,
+		cfg.UsageReportConfig.InsecureSkipVerify,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	return &runnables.Leader{
+		Runnable: runnables.NewCronJob(runnables.CronJobConfig{
+			Worker:       usage.CreateUsageJobWorker(logger, k8sClient, reporter, cfg),
+			Logger:       logger,
+			Period:       cfg.TelemetryReportPeriod,
+			JitterFactor: telemetryJitterFactor,
+			ReadyCh:      readyCh,
+		}),
+	}, nil
+}
+
+func createUsageWarningJob(cfg config.Config, readyCh <-chan struct{}) *runnables.Leader {
+	logger := cfg.Logger.WithName("usageReporter")
+	worker := func(_ context.Context) {
+		logger.Error(
+			errors.New("usage reporting not enabled"),
+			"Usage reporting must be enabled when using NGINX Plus; redeploy with usage reporting enabled",
+		)
+	}
+
+	return &runnables.Leader{
```

Comment:
would it make sense to have a job that runs on all replicas? This way every replica will get warnings